cmd/observe/flows.go

```diff
@@ -46,6 +46,8 @@ import (
 var verdicts = []string{
 	flowpb.Verdict_FORWARDED.String(),
 	flowpb.Verdict_DROPPED.String(),
+	flowpb.Verdict_AUDIT.String(),
```

Comment:
This is not something that was added in the PR. I wonder why it was never added

Reply:
> This is not something that was added in the PR.

I'm not sure what you mean. This was added on purpose by the second commit in this pull request.

> I wonder why it was never added

I assumed it was an oversight. I don't see a reason we would want to keep this hidden.

Reply:
I didn't look at the commit basis at first. Yes, this is fine to commit as is, I just thought it was squashed all in one.